helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -378,6 +378,11 @@ agent:
   # Applicable only when updateStrategy is set to "RollingUpdate".
   # Can be an absolute number or a percentage. The default is 1.
   maxUnavailable: 1
+  
+  service:
```

Comment:
Please add new variable to the json schema here https://github.com/signalfx/splunk-otel-collector-chart/blob/main/helm-charts/splunk-otel-collector/values.schema.json